.github/ISSUE_TEMPLATE/joint-review.md

```diff
@@ -23,10 +23,10 @@ Security Provider: yes/no (e.g. Is the primary function of the project to suppor
    - [ ] Project security lead
    - [ ] Lead security reviewer
    - [ ] 1 or more additional reviewer(s)
-   - [ ] Every reviewer has read [security reviewer guidelines](https://github.com/cncf/sig-security/blob/master/assessments/guide/security-reviewer.md) and stated declaration of conflict
+   - [ ] Every reviewer has read [security reviewer guidelines](https://github.com/cncf/tag-security/blob/main/assessments/guide/security-reviewer.md) and stated declaration of conflict
    - [ ] Sign off by 2 chairs on reviewer conflicts
 - [ ] Create slack channel (e.g. #sec-assess-projectname)
-- [ ] Project lead provides draft document - see [outline](https://github.com/cncf/sig-security/blob/master/assessments/guide/outline.md)
+- [ ] Project lead provides draft document - see [outline](https://github.com/cncf/tag-security/blob/main/assessments/guide/outline.md)
```

Comment:
This is a 404. But It also was a 404 before the change.

Reply:
@lumjjb @TheFoxAtWork should this be https://github.com/cncf/tag-security/blob/main/assessments/guide/self-assessment.md ?

Reply:
I didn't see the file, so I don't think the link I provided is correct.  Wrote up issue: https://github.com/cncf/tag-security/issues/650